app/views/static/about.haml

```diff
@@ -1,67 +1,31 @@
-- provide(:title, generate_title('About'))
 .jumbotron
   .jumbotron__content
-    %h1= APP_CONFIG['site_name']
-    %p= t 'views.about.subtitle'
+    %h1= APP_CONFIG["site_name"]
+    %p= t(".subtitle")
 .container
-  .card
-    .card-body
-      .row
-        .col-md-4
-          %h3= t 'views.about.links.title'
-          %p= t('views.about.links.desc', app_title: APP_CONFIG['site_name'])
-        .col-md-4.col-sm-6
-          %a{ href: 'https://twitter.com/retrospring' }
-            .icon--showcase
-              %i.fa.fa-twitter
-          %h4.heading-about.text-center
-            Twitter
-        .col-md-4.col-sm-6
-          %a{ href: help_faq_path }
-            .icon--showcase
-              %i.fa.fa-question
-          %h4.heading-about.text-center
-            FAQ
-  .card
-    .card-body
-      .row
-        .col-md-4
-          %h3= t 'views.about.opensource.title'
-          %p= t('views.about.opensource.warning', app_title: APP_CONFIG['site_name'])
-          %p= raw t('views.about.opensource.desc',
-            app_title: APP_CONFIG['site_name'],
-            github: link_to(t('views.about.opensource.github'), 'https://github.com/Retrospring/retrospring'),
-            bugtracker: link_to(t('views.about.opensource.bugtracker'), 'https://github.com/Retrospring/retrospring/issues'))
-        .col-md-4
-          %a{ href: 'https://github.com/Retrospring/retrospring' }
-            .icon--showcase
-              %i.fa.fa-github
-          %h4.heading-about.text-center= t 'views.about.repository.title'
-          %p.text-center
-            %em= t 'views.about.repository.desc'
-  .card
-    .card-body
-      .row
-        .col-md-3.col-sm-12.col-xs-12
-          %h3= t 'views.about.statistics.title'
-          %p= t('views.about.statistics.desc', app_title: APP_CONFIG['site_name'])
-        .col-md-3.col-sm-6.col-xs-6
-          .entry.entry--statistics
-            %h2.entry__value#asked-count= @questions
-            %h4.entry__description= t('views.general.question').pluralize(@questions)
-          .entry.entry--statistics
-            %h2.entry__value#answered-count= @answers
-            %h4.entry__description= t('views.general.answer').pluralize(@answers)
-        .col-md-3.col-sm-6.col-xs-6
-          .entry.entry--statistics
-            %h2.entry__value#comment-count= @comments
-            %h4.entry__description= t('views.general.comment').pluralize(@comments)
-          .entry.entry--statistics
-            %h2.entry__value#smile-count= @smiles
-            %h4.entry__description= t('views.general.smile').pluralize(@smiles)
-        .col-md-3.col-sm-12.col-xs-12
-          .entry.entry--users
-            .entry__value#follower-count= @users
-            %h4.entry__description= t('views.general.user').pluralize(@users)
+  .row
+    .col-sm-8
+      .card.card--fullheight
+        .card-body
+          - if APP_CONFIG["about"].present?
```

Comment:
I’m sure we’ll forget to set this before the next deploy ;-)

Reply:
It'll just show

> The site owner decided not to describe this Retrospring instance, how mysterious!

For a while then :trollface: